package.json

```diff
@@ -74,7 +74,7 @@
     "fs-extra": "^7.0.1",
     "glob": "^7.1.2",
     "is-docker": "^1.1.0",
-    "jest": "^24.7.1",
+    "jest": "^27.2.1",
```

Comment:
This fixes the next issue with `cwd` not being found. `jest` and `babel-jest` need to be on the same version.

https://github.com/facebook/jest/issues/7868#issuecomment-463642113